include/yokozuna.hrl

```diff
@@ -263,6 +263,9 @@
 -define(YZ_ENTROPY_LOCK_TIMEOUT,
     app_helper:get_env(?YZ_APP_NAME, anti_entropy_lock_timeout, 10000)).
 
+-define(YZ_ENABLE_DIST_QUERY,
```

Comment:
I know this pattern is already being used in other places in Yokozuna, but I don't like it. Defines without any arguments like this are normally used for constants, so when I see places where something like this is used, I wouldn't normally expect it to expand out to a function call that dynamically fetches the current config.

Plus we only use it in two places anyway, so it feels like we might as well just inline the app_helper:get_env call directly in the code.